app/models/vacancy.rb

```diff
@@ -294,13 +296,10 @@ def find_conflicting_vacancy
     find_conflict_vacancy || find_duplicate_vacancy
   end
 
+  # soft delete so that all the stats etc are kept even after the vacancy no longer exists
   def trash!
-    return if trashed?
```

Comment:
Question here:
wouldn't be better to keep the guard clause here?
`return if discarded?` 

If not multiple calls over `trash!` would repeatedly triggering the document purge job and google index removal API call, and I imagine also updating the discarding timestamp?

Reply:
I wasn't sure whether the scenario was even possible - its only the unit test that keeps the code executing. Now we only load  'kept' vacancies, it seems unlikely that double-trashing could ever take place